server/@types/bapv.d.ts

```diff
@@ -38,10 +38,11 @@ export type BookingJourney = {
   visitorSupport?: string
 
   // main contact
-  mainContact?: {
-    contact: Visitor | string // either known Visitor or 'other' name
-    phoneNumber?: string
-  }
+  mainContact?: Visitor | string // either known Visitor or 'other' name
```

Comment:
Splitting this data in the session because it is now collected independently on different pages.